cpp/src/cider/function/scripts/parse_ast.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import print_function
```

Comment:
when will this be used?

Reply:
This is used for external function like `between` generation from header file. Used in CMake file.